### DISCUSSION TITLE
一块钱买来的自尊心

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#289] [Announcements]
https://bbing.com.cn/202206/1yzzx/

**Comment by xinjiawei** [comment]
写得好，可以转载一下吗，会标明出处的

**Comment by caibingcheng** [comment]
> @xinjiawei
> 写得好，可以转载一下吗，会标明出处的

可以转载, 遵循[CC BY-NC 4.0](https://creativecommons.org/licenses/by-nc/4.0/deed.zh-Hans)即可

**Comment by W4J1e** [comment]
想起小时候去上学，每天早上都要走接近一个小时的路，从山的这头到另一座山的那头。下雨天，家里只有一把雨伞，我姐就披着塑料袋去上学，所以相比而言我常常比我姐幸福太多。

**Comment by caibingcheng** [comment]
> @W4J1e
> 想起小时候去上学，每天早上都要走接近一个小时的路，从山的这头到另一座山的那头。下雨天，家里只有一把雨伞，我姐就披着塑料袋去上学，所以相比而言我常常比我姐幸福太多。

羡慕有亲兄弟姐妹的~~(我小时候奶奶还会给我伞上绣上名字)

**Comment by penndu** [comment]
摸爬滚打这么多年，发现父母很多经验都是对的，并没有因为观念旧过时！

**Comment by wellsleep** [comment]
每个人都有自己的看法，可能当时父母有他们的考量，而孩子的思考往往非常简单和单纯。我有一个一岁多的孩子，她某些时候一定也会误解我，但是我愿意默默给她提供最好的条件。

**Comment by caibingcheng** [comment]
> @penndu
> 摸爬滚打这么多年，发现父母很多经验都是对的，并没有因为观念旧过时！

是啊，父母或者老人有很多经验都挺对的，但是我有时候就喜欢对抗父母，然后在社会上经历了或者吃了大亏，才会知道他们说的没错

**Comment by caibingcheng** [comment]
> @wellsleep
> 每个人都有自己的看法，可能当时父母有他们的考量，而孩子的思考往往非常简单和单纯。我有一个一岁多的孩子，她某些时候一定也会误解我，但是我愿意默默给她提供最好的条件。

父母肯定都是为了孩子好的，但是也要意识到自身的时代局限性，比如上学时孩子的社交圈/刚步入社会时的工作圈等等，因为时代和环境的不同，这些都可能成为父母的时代局限性（我还没孩子，但是也见过一些父母喜欢以“我们那时候怎么怎么样”来要求和教育孩子），在这些时候可能适当的放开/听取孩子的意见会好一点...（现在回想自己小时候很容易就顶撞父母就觉得我有点过分~）

**Comment by goodpaperman** [comment]
一块钱不少了，能坐三轮为什么非要坐车呢，想不通

**Reply by goodpaperman** [reply]
给你讲一个我的故事，小时候家附近开了一个游乐场，临时性质的，开几天就去别处那种。别人给了老爸几张票，他带我去里面玩，其中有一张是卡丁车。那会儿没玩过呀，特别过瘾，玩了一圈该下车了，老爸看我挺高兴，就和管理员说再来一圈。小地方的人嘛，没那么多束缚，觉得一圈就要好多圈不值，买了票再来一圈也合理，但是这个外地来的管理员硬不放行，就把整个队列都堵在那里了。好多人都看向我们，我觉得好尴尬，感觉影响到其它人了，老爸还在那里争，我脸面上过不去，就主动下车走了，老爸也很生气，他还在那里争取了我先反水了，两人悻悻的回去了。
我觉得我爸的行为就损伤了我的自尊，我不希望做那种贪图便宜的人，喜欢玩可以掏钱再买一张票，而不是用这种方式，还影响到了周围的人。相比你说的这种情况，其实还好，毕竟坐着自己的三轮去哪里，都没违法公众良俗，对其它人也没影响。

**Reply by caibingcheng** [reply]
我现在这个年纪就不太关心这些了。
小孩对这些事情本来是没有什么尴尬、不好意思之类情绪的，但是很容易受到环境影响。
比如同学们经常说你穷、说你丑，买包辣条都会很惊讶（“你居然有钱买辣条”），还有本来坐在三轮车上和父亲开开心心卖水果的时候，周围就会有些小孩看你指指点点的。长期在这种环境下，加上没人及时开导，难免就容易有这些“不好意思”的心理。
